.templates/auto-update.sh.template

```diff
@@ -0,0 +1,21 @@
+{% extends "templates/base_script.sh.template" %}
+
+{% block script %}
+    COMMIT_MSG="$(git log -1 --pretty=%B)"
```

Comment:
This shows the full body for the last commit. Another option is to show the commit hashes and subjects for something like the last 3 commits (`git log -3 --pretty='%h - %s'`). I'm thinking that a) some PRs, like this one, have more than one commit, and b) we don't necessarily need the full body from the Nengo Bones commit in every downstream repo.

Reply:
I suspect anyone who really wants to know what has changed is just going to go look at the Nengo Bones commit history/changelog, so I don't think it matters too much. This was just kind of "well it's easy to do and doesn't hurt, so might as well". But `git log -3 --pretty="%h - %s"` is equally easy, so doesn't matter to me.